templates/cicd/buildspec.yml

```diff
@@ -23,6 +23,7 @@ phases:
       # First, upgrade the cloudformation stack of every environment in the pipeline.
       - pipeline=$(cat $CODEBUILD_SRC_DIR/copilot/pipeline.yml | ruby -ryaml -rjson -e 'puts JSON.pretty_generate(YAML.load(ARGF))')
       - pl_envs=$(echo $pipeline | jq '.stages[].name' | sed 's/"//g')
+      - ./copilot-linux app upgrade
```

Comment:
Does this work? I don't see CloudFormation permissions for the BuildRole in the pipeline

Reply:
Yes you are right. We need to add more permissions to the role. Addressed

Reply:
How's the migration story? Am I right in assuming that it's: 
1) Download new version of Copilot.
2) Manually edit the buildspec to include `app upgrade`. 
3) Run `copilot pipeline update` to propagate the new build role changes.
3) Push the buildspec to source.

How many manual interventions are needed?